formulaic/transforms/hashed.py

```diff
@@ -24,9 +24,9 @@ def hashed(
     spans_intercept: bool = False,
 ):
     """
-    Hashes the values of a factor into a fixed number of levels. DUe to the
-    large number of levels required to avoid collisions, this transform should
-    be used with output='sparse' to avoid memory issues.
+    Deterministically the values of a factor into a fixed number of levels.
```

Comment:
nit: whoops. You dropped "hashes". Should be: "Deterministically hashes..."

Reply:
Good catch, so I did. Fixed.